README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 | Java / Groovy | Yes                | Yes                |
 | C++           | No                 | Yes                |
 | JavaScript    | No                 | Yes                |
-| gRPC          | -                  | Yes                |
-
-## Run Deephaven
-
-This section is a quick start guide for running Deephaven from pre-built images.  Almost all users will want to run Deephaven using pre-built images.  It is the easiest way to deploy.  For detailed instructions, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart).
-
-Developers interested in tinkering with and modifying source code should build from the source code. For detailed instructions on how to do this, see [Build and launch Deephaven](https://deephaven.io/core/docs/how-to-guides/launch-build).
-
-If you are not sure which of the two is right for you, use the pre-built images.
-
-### Required Dependencies
-
-Running Deephaven requires a few software packages.
-
-| Package        | Version                       | OS           |
-| -------------- | ----------------------------- | ------------ |
-| docker         | ^20.10.8                      | All          |
-| docker-compose | ^1.29.0                       | All          |
-| Windows        | 10 (OS build 20262 or higher) | Only Windows |
-| WSL            | 2                             | Only Windows |
-
-You can check if these packages are installed and functioning by running:
-```
-docker version
-docker-compose version
-docker run hello-world
-```
-
-> :warning: **On Windows, all commands must be run inside a WSL 2 terminal.**
+| Go            | No                 | Yes                |
+| R             | No                 | Yes                |
 
-If any dependencies are missing or unsupported versions are installed, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart#prerequisites) for installation instructions.
+## Installation
```

Comment:
This section is not going to be clear to users coming to this document.  It kind of machine guns client and server concepts and then lumps client and server links together.  It needs some thought.

Reply:
I'm not going to close this comment, since I'm not sure how I should restructure this section.